transaction_integration_test.go

```diff
@@ -517,183 +475,6 @@ func TestTrxPaymentMethodNonce(t *testing.T) {
 	}
 }
 
-func TestTransactionCreateSettleAndFullRefund(t *testing.T) {
```

Comment:
I think we should leave this function and the one below it here, because they are primarily concerned with testing transaction refund functionality, and only use the test-only settle endpoint to get the transaction into a state where it can be refunded.

I agree it makes sense to move the other two tests that are in this file since they seem primarily concerned with testing that the test-only settle functionality works.